packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { unstable_composeClasses as composeClasses } from '@mui/utils';
-import Popper from '@mui/material/Popper';
-import NoSsr from '@mui/material/NoSsr';
+import { Popper } from '@mui/base/Popper';
```

Comment:
It could be interesting to test the https://mui.com/base-ui/react-popup/. cc @michaldudak

Reply:
Indeed! Please do if you can!